packages/jss/tests/unit/escape.js

```diff
@@ -1,35 +1,16 @@
-/* eslint-disable no-underscore-dangle */
-
 import expect from 'expect.js'
 import escape from '../../src/utils/escape'
 
-const originalEscape = CSS.escape
-
 describe('Unit: jss - escape', () => {
   describe('with CSS.escape', () => {
     it('should escape in development', () => {
       expect(escape('test()')).to.be('test\\(\\)')
     })
 
     it('should not escape in production', () => {
-      escape.__Rewire__('env', 'production')
+      process.env.NODE_ENV = 'production'
       expect(escape('test()')).to.be('test()')
-      escape.__ResetDependency__('env')
-    })
-  })
-
-  describe('without CSS.escape', () => {
```

Comment:
I don't think we need to test this here. We just need to make sure that it works in our supported browsers.

Reply:
I think this test is there to ensure the regex fallback works.

Reply:
since in most env it will use CSS.escape